deploy/olm-catalog/smart-gateway-operator/1.0.0/smart-gateway-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -38,7 +38,7 @@ metadata:
     certified: "false"
     containerImage: quay.io/infrawatch/smart-gateway-operator:v1.0.0
     createdAt: "2019-11-14T14:49:00Z"
-    description: Smart Gateway Operator
+    description: Operator for managing the Smart Gateway Custom Resources, resulting in deployments of the Smart Gateway.
```

Comment:
Before merging this will need to be updated to whatever the latest CSV version is.